packages/components/pages/f-registration/test/component/f-registration.component.spec.js

```diff
@@ -19,4 +19,18 @@ describe('Shared - f-registration component tests', () => {
         // Assert
         expect(registration.canBeClicked(link)).toBe(true);
     });
+
+    forEach([
+        ['termsAndConditionsLink', '/info/terms-and-conditions'],
+        ['privacyPolicyLink', '/info/privacy-policy'],
+        ['cookiesPolicyLink', '/info/cookies-policy']
+    ])
+    .it('should check if the legal documentation is clickable', (linkName, expectedUrl) => {
```

Comment:
the test looks good, just wonder if the title is a little misleading as we're testing that the resolved URL is a match?

Reply:
Ah sorry, good catch, copy and paste error